src/bindings/c/include/openvino/c/ov_node.h

```diff
@@ -52,7 +52,7 @@ OPENVINO_C_API(ov_status_e) ov_node_get_shape(ov_output_const_node_t* node, ov_s
  * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_node_list_get_any_name_by_index(ov_output_node_list_t* nodes, size_t idx, char** tensor_name);
+ov_node_list_get_any_name_by_index(const ov_output_node_list_t* nodes, size_t idx, char** tensor_name);
```

Comment:
I still have a question, why don't we simplify such methods to:
```
ov_port_get_any_name(const ov_output_node_t* node, char** tensor_name);
ov_port_get_shape(const ov_output_node_t* node, ov_shape_t* shape);
...
```

Reply:
It is handled by https://github.com/openvinotoolkit/openvino/pull/12617